app/assets/javascripts/tylium/modules/sidebar.js

```diff
@@ -6,50 +6,55 @@
     this.$viewContent = $sidebar.siblings('[data-behavior~=view-content]');
     this.storageKey = $sidebar.data('storage-key');
 
+    // Must match $sidebarCollapsed and $sidebarExpanded values from tyluim/variables.scss
```

Comment:
Is there a way we can calculate this from the view/CSS so we don't have to set this in multiple places?

Reply:
Yep, made this change. Thanks.